index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Generate by [js2dts](https://github.com/whxaxes/js2dts#readme)
-
+// Generate by [js2dts](https://github.com/whxaxes/js2dts#readme)
+

```

Comment:
What is the difference here?

Reply:
No idea, i dont know which dependency generated the sentence